src/components/ApplicationDashboard.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import styled from 'styled-components'
+import { H1 } from './Typography'
+import { Button } from './Input/Button'
+import { SOCIAL_LINKS } from '../utility/Constants'
+import facebook from '../assets/icons/facebook.svg'
+import instagram from '../assets/icons/instagram.svg'
+import medium from '../assets/icons/medium.svg'
+import twitter from '../assets/icons/twitter.svg'
+import { ReactComponent as HandWave } from '../assets/hand-wave.svg'
+
+const Container = styled.div`
+  margin: 5em auto;
+  width: 70%;
+`
+
+const WelcomeHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 1em;
+`
+
+const WelcomeMessage = styled(H1)`
+  text-align: center;
+`
+
+const StyledHandWave = styled(HandWave)`
+  margin-left: 10px;
+  margin-top: 20px;
+`
+
+const AppLinks = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const HackerAppText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  margin-top: 1.55em;
+`
+
+const EditAppButton = styled(Button)`
+  width: 250px;
+  margin-right: 0;
+`
+
+const StatusContainer = styled.div`
+  min-height: 350px;
+  padding: 2.5em 4.5em 2.5em 3em;
+  border-radius: 21px;
+  background-color: ${p => p.theme.colors.applicationCard};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const AppStatusText = styled.p`
+  font-size: 1.25em;
+  font-weight: 700;
+  margin-top: 0;
+`
+
+const StatusBlurbText = styled.p`
+  color: ${p => p.theme.colors.primary};
+  font-weight: 700;
+  line-height: 2em;
+  margin-top: 0.75em;
+`
+
+const FooterContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: -24px;
+`
+
+const SocialMediaIcons = styled.img`
+  margin: 24px;
+`
+
+const RSVPButton = styled(Button)`
+  width = 100px;
+  margin-right: 0;
```

Comment:
```suggestion
  margin-right: 0;
  ${maxWidthMediaQueries('mobile')} {
margin: 1em;
  }
```
to center the button in mobile (see one of my above screenshots)